src/pages/K8s.vue

```diff
@@ -23,6 +23,10 @@
       @updateCPU="handleUpdateCPU"
       @warning="handleWarning"
     />
+    <div id="portWrapper" class="labeled-input">
+      <LabeledInput :value="settings.kubernetes.port" :label="'Port'" type="'number'" @input="handleUpdatePort" />
```

Comment:
```suggestion
      <LabeledInput :value="settings.kubernetes.port" label="Port" type="number" @input="handleUpdatePort" />
```
- No need to make JS evaluate the string `'Port'`
- The type needs to _not_ be double-quoted; you should not be able to type letters into that textbox.

Reply:
Done